tools/ete/ete_init_taxdb.py

```diff
@@ -0,0 +1,23 @@
+import ete3.ncbi_taxonomy
+import optparse
```

Comment:
import optparse should be before ete3.ncbi_taxonomy

Reply:
Yep, we (as `galaxy` and `tools-iuc`) use the "smarkets" style, which divides imports in 4 groups (separated by a single blank line):

- `from __future__ import ...`
- standard library imports
- third-party imports
- application-specific imports

Within each grouping, imports should be sorted lexicographically, ignoring case, according to each module's full package path, but with `import ...` statements before `from X import ...` statements, see an [example](https://github.com/PyCQA/flake8-import-order/blob/master/tests/test_cases/complete_smarkets.py).

So, in this case it should be:

```python
import optparse

import ete3.ncbi_taxonomy
from six.moves.urllib.request import urlretrieve
```